### PR TITLE
Docs: Refactors notifications in alerting

### PR DESCRIPTION
**Pull request by brendamuir**
Refactors and creates a manage alert notifications section - moves contact points and notification policies to this section. 
Also moves conceptual information on those topics up to the fundamentals/explore section.

### CODE REVIEW

File: docs/sources/alerting/manage-notifications/create-contact-point.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  - templating
+title: Manage contact points
+weight: 100
+---
+
+# Manage contact points
+
+Use contact points to define how your contacts are notified when an alert rule fires. You can create, edit, delete, and test a contact point.
+
+To add a contact point:
```
Comment (chri2547) [inline docs/sources/alerting/manage-notifications/create-contact-point.md:25]: this should be an H2 `Add a contact point` with a short intro, before you begin section (optional), stem sentence (optional), and then the steps.

File: docs/sources/alerting/manage-notifications/create-contact-point.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+1. Click **New contact point**.
+1. From the **Alertmanager** dropdown, select an Alertmanager. By default, Grafana Alertmanager is selected.
+1. In **Name**, enter a descriptive name for the contact point.
+1. From **Contact point type**, select a type and fill out mandatory fields. For example, if you choose email, enter the email addresses. Or if you choose Slack, enter the Slack channel(s) and users who should be contacted.
+1. Some contact point types, like email or webhook, have optional settings. In **Optional settings**, specify additional settings for the selected contact point type.
+1. In Notification settings, optionally select **Disable resolved message** if you do not want to be notified when an alert resolves.
+1. To add another contact point type, click **New contact point type** and repeat steps 6 through 8.
+1. Click **Save contact point** to save your changes.
+
+To edit a contact point:
```
Comment (chri2547) [inline docs/sources/alerting/manage-notifications/create-contact-point.md:38]: same here

File: docs/sources/alerting/manage-notifications/create-contact-point.md (hunk 1 above)
Comment (brendamuir) [reply docs/sources/alerting/manage-notifications/create-contact-point.md:25]: thanks Chris will update

File: docs/sources/alerting/manage-notifications/create-contact-point.md (hunk 2 above)
Comment (chri2547) [reply docs/sources/alerting/manage-notifications/create-contact-point.md:38]: here's an example of how I format multi-task topics.
https://grafana.com/docs/grafana/latest/administration/team-management/

**Review by gillesdemey — APPROVED** [review]

File: docs/sources/alerting/manage-notifications/_index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+As a first step, define your contact points; where to send your alert notifications to. A contact point can be a set of destinations for matching notifications. Add message templates to contact points for reuse and consistent messaging in your notifications.
+
+Next, create a notification policy which is a set of rules for where, when and how your alerts are routed to contact points. In a notification policy, you define where to send your alert notifications by choosing one of the contact points you created.Add mute timings to your notification policy. A mute timing is a recurring interval of time during which you don’t want any notifications to be sent out.
+
+You can also add silences to stop notifications from one or more alert rules. The difference between a silence and a mute timing is that a silence only lasts for only a specified window of time.
+When an alert rule fires, the alert ruler sends alert instances to the Alertmanager; one alert rule can trigger multiple individual alert instances.
```
Comment (joeblubaugh) [inline docs/sources/alerting/manage-notifications/_index.md:22]: Should there be a blank line separating paragraphs?

**Review by joeblubaugh — APPROVED** [review]

File: docs/sources/alerting/manage-notifications/_index.md (hunk 3 above)
Comment (brendamuir) [reply docs/sources/alerting/manage-notifications/_index.md:22]: yes--thanks i'll add that :)

**Pull request merged**